cv32e40x/regress/cv32e40x_full.yaml

```diff
@@ -69,213 +69,237 @@ builds:
     cfg: pma_test_cfg_5
     dir: cv32e40x/sim/uvmt
 
+  uvmt_cv32e40x_no_bitmanip:
+    cmd: make comp_corev-dv comp
+    cfg: no_bitmanip
+    dir: cv32e40x/sim/uvmt
+
 # List of tests
 tests:
   hello-world:
     build: uvmt_cv32e40x
     description: uvm_hello_world_test
+    builds: [ uvmt_cv32e40x, uvmt_cv32e40x_no_bitmanip]
```

Comment:
Why does it look like "no_bitmanip" is set for almost every test? Is the intention that all of these shall run with no_bitmanip, or is there a mechanism on top of this which ensures the separation of "daily" and "weekly" regressions?

Reply:
This means run default _and_ no_bitmanip

Reply:
I did a small test run with a full regression, and decided as of this commit to not split out a weekly regression yet, the increase of tests is not that big - this might still be a consideration for the future though

Reply:
Ok. I was just confused because the PR description said _"Expanded full regression to include **a few** non-bitmanip enabled tests"_ but this seemed like more than a few. If the impact in run-time was not too bad then I agree that this is fine.